deployment/ops/ccip/seq_deploy_and_init_ccip.go

```diff
@@ -251,6 +251,19 @@ var DeployAndInitCCIPSequence = cld_ops.NewSequence(
 			return DeployCCIPSeqOutput{}, err
 		}
 
+		// init transfer ownership to MCMS
+		_, err = cld_ops.ExecuteOperation(
```

Comment:
Every CCIP sequence inits the ownership transfer to MCMS. The next step is to execute the [MCMS accept ownership proposal](https://github.com/smartcontractkit/chainlink-sui/pull/116/files#diff-2455dc057ea7d70c3262ebcfb748b7edd946f2c679000eceb8c055cb9406366cR58)